src/js/player.js

```diff
@@ -3341,6 +3341,40 @@ class Player extends Component {
   }
 
   /**
+   * Reports whether or not a player has a plugin available.
+   *
+   * This does not report whether or not the plugin has ever been initialized
+   * on this player. For that, [usingPlugin]{@link Player#usingPlugin}.
+   *
+   * @param  {string}  name
+   *         The name of a plugin.
+   *
+   * @return {boolean}
+   *         Whether or not this player has the requested plugin available.
+   */
+  hasPlugin(name) {
+    // While a no-op by default, this method is created in plugin.js to avoid
```

Comment:
we can probably use the same trick that @BrandonOCasey has been using where we just have the doc comment but no actual function.
I wonder if we can specify partials for jsdoc.